INSTALL.linux.rst

```diff
@@ -132,3 +132,17 @@ Development tools
 #. Make migrations::
 
     make migrations
+
+#. Run Dockerized Node
```

Comment:
there is special run section. we distinguish them, because you configure once, but run many